content/news/speaker-support.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Speaker Support for PyLadiesCon 2024"
+meta_title: "Speaker Support for PyLadiesCon 2024"
+description: "We have speaker support for those of you who need a bit more reassurance and encouragement!
+"
+date: 2024-08-28T05:00:00Z
+categories: ["Blog Post",]
+author: "Cheuk"
+tags: ["python", "community", "conference", "public speaking", "pyladies"]
+draft: false
+layout: news
+---
+
+Since there are only 2 weeks until the [Call for Propocal](https://pretalx.com/pyladiescon-2024/cfp) closes, we do not want to you miss this speaking opportunity. Do you still need some encouragement in submitting a proposal? Maybe you need a bit more help? Don't worry! We have speaker support for those of you who need a bit more reassurance and encouragement!
```

Comment:
```suggestion
Since there are only 2 weeks until the [Call for Proposal](https://pretalx.com/pyladiescon-2024/cfp) closes, we do not want to you miss this speaking opportunity. Do you still need some encouragement in submitting a proposal? Maybe you need a bit more help? Don't worry! We have speaker support for those of you who need a bit more reassurance and encouragement!
```